src/ln/msgs.rs

```diff
@@ -585,6 +586,9 @@ pub trait RoutingMessageHandler : Send + Sync {
 	fn handle_channel_update(&self, msg: &ChannelUpdate) -> Result<bool, HandleError>;
 	/// Handle some updates to the route graph that we learned due to an outbound failed payment.
 	fn handle_htlc_fail_channel_update(&self, update: &HTLCFailChannelUpdate);
+	///Gets a subset of the messages required to dump our routing table to a remote node, starting at the short_channel_id indicated by starting_point and including batch_amount entries
+	/// This function will start iterating at 0 if the starting_point is < 0.
```

Comment:
So why not just make it a u64? It's converted to a u64 in the impl anyway.

Reply:
Should be u64, tried something that didnt work out here.